vueui/src/components/view/misc/ghostspawner.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div>
+    <div class="tagselector" v-show="!spawnpoint">
+      <vui-button v-for="(amount,tag) in tags" :key="tag" v-on:click="current_tag = tag" v-bind:class="{ selected: current_tag == tag}">{{tag}} ({{amount}})</vui-button>
+    </div>
+    <hr v-show="!spawnpoint">
```

Comment:
Why this 'seperator' showing depends if spawnpoint is set or it is not set?

Reply:
So there isnt a random horizontal line at the top if the ui for a specific spawnpoint is shown.